pkgs/by-name/ff/ffts/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  stdenv,
+  lib,
+  cmake,
+  git,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ffts";
+  version = "unstable-2019-03-19";
+
+  src = fetchFromGitHub {
+    owner = "linkotec";
+    repo = "ffts";
+    rev = "2c8da4877588e288ff4cd550f14bec2dc7bf668c";
+    hash = "sha256-Cj0n7fwFAu6+3ojgczL0Unobdx/XzGNFvNVMXdyHXE4=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    git
```

Comment:
```suggestion
```
Git is not required. The cloning happends before the build, and the build does not have Internet access.
Also remember to remove `git` from the derivation inputs.